windows/CMakeLists.txt

```diff
@@ -30,9 +30,9 @@ if(NOT NUGET)
   endif()
 endif()
 
-add_custom_target(DEPENDENCIES_DOWNLOAD ALL)
+add_custom_target(DEPENDENCIES_DOWNLOAD_jnschulze ALL)

```

Comment:
Please use ${PROJECT_NAME} as a prefix for the target name.

Reply:
Ok, done.

After merge, can you publish in pub.dev? You use fmt submodule, so pubspec.yaml cannot recognize submodule if we point dependency to git url. The only way is point to my local path, but making this, my CI (Codemagic) not work.

If you quicky publish on pub.dev we can imediatly use this fix.

Regards

Reply:
Yup, I've just published `v.0.1.8` on pub.dev.